### PR TITLE
Installer prereq checks

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by aidenkeating** [comment]
@david-martin @jasonmadigan Would you mind taking a look?

**Comment by jasonmadigan** [comment]
@aidenkeating will give this a spin on a fresh fedora vm to see how I get on.

File: installer/install.sh [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+}
+
+function check_passed_msg() {
+  echo "✓ ${1} check passed."
+}
+
+function check_docker() {
+  docker_version=$(docker version --format '{{json .Client.Version}}')
+  if [[ $docker_version == *"-rc"* ]]; then
+    echo "Docker version is not good. Use latest Stable release"
```
[the remark this answers is not in the document]
Comment (aidenkeating) [reply installer/install.sh]: Yeah this bit needs an update, I'll go with something along those lines

**Comment by david-martin** [comment]
@aidenkeating This gets my vote :+1: 
I tried it locally and works great!
However, I have been installing/cleaning many times, so my system meets the pre-reqs. 
We'll have to fix/adapt as cases come through where the pre-reqs don't pick up on some things.

The only thing I'll suggest is updating the local-setup guide to say to run this script instead.
Sound reasonable?

**Comment by jasonmadigan** [comment]
@aidenkeating one miss: npm/node check

**Comment by aidenkeating** [comment]
@david-martin I've made a small update to the f2f doc to say to use the `0.0.6` tag when prompted. Other than that everything is linked to the local setup doc. Also added a check for NPM. I think we're ready to merge?

**Pull request merged**